swig/python/CMakeLists.txt

```diff
@@ -261,13 +285,14 @@ if (VISUALISATION)
 endif()
 
 # Configure and build the python package
-# The file configurations above will already have output setup.py and __init__.py to the appropriate places and created the directory for the module
+# The file configurations above will already have output setup.py, __init__.py and the code generator module files to the appropriate places and created the directory for the module
 add_custom_command(TARGET ${PYTHON_MODULE_NAME} POST_BUILD
 	# copy generated python file to library output location
 	COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_LIB_TEMP_DIRECTORY}/${PYTHON_MODULE_NAME}/${PYTHON_MODULE_NAME}.py ${PYTHON_MODULE_NAME}
 	# copy the compiled pyd file to library output location
 	COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:${PYTHON_MODULE_NAME}> ${PYTHON_MODULE_NAME}
 	COMMAND ${Python3_EXECUTABLE} setup.py bdist_wheel
+  DEPENDS ${PYTHON_CODEGEN_SRC_FILES}
```

Comment:
This may also be insufficient as above, due to it being a post_build event so it doesn't always fire. Not 100% though.

Reply:
80% of the time it works every time!

Reply:
maybe resolved by https://github.com/FLAMEGPU/FLAMEGPU2/pull/887

Reply:
Resolved by PR #887 now merged into this branch